### DISCUSSION TITLE
Page custom livewire

### DISCUSSION THREAD

**Discussion by Wellarts** [filamentphp/filament#5560] [Help]
I created a custom livewire page, how do I add it to open within the administration panel and only logged in user?

**Comment by ArtMin96** [comment]
did you mean filament page?

**Reply by Wellarts** [reply]
I believe that the filament page would be the resource, the custom page I created was with the command php artisan make:livewire NomePage

**Reply by danharrin** [reply]
`make:filament-page` doesn't need to belong to a resource
